NAMESPACE

```diff
@@ -60,6 +60,9 @@ export(ui_todo)
 export(ui_value)
 export(ui_warn)
 export(ui_yeah)
+export(use_action_ci_full)
```

Comment:
I think it would better to use `check` here rather than `ci`, or maybe even `rcmdcheck`, because eventually we'll want `use_action_bookdown()`, `use_action_blogdown()` etc

Reply:
Ok I did that in 9ac19994

Reply:
hi, where do we find `use_action_bookdown()` or `use_action_blogdown()`? @murraycadcow @jimhester ?

Reply:
They don't exist currently